cli/src/test/java/com/box/l10n/mojito/cli/command/TMImportCommandTest.java

```diff
@@ -135,11 +137,15 @@ public void testImportWithImportExportNote() throws Exception {
 
     tmTextUnitVariant = iterator.next();
     assertEquals("1時間", tmTextUnitVariant.getContent());
-    assertEquals(new DateTime(1447198865000L), tmTextUnitVariant.getCreatedDate());
+    assertEquals(
+        ZonedDateTime.ofInstant(Instant.ofEpochMilli(1447198865000L), ZoneId.systemDefault()),
```

Comment:
same as earlier, maybe `AtZone()` is better? 

There is no equivalent to the old constructor? the old construct was way shorter? 

Should we introduce a utility class that does that? what other project do?